server/start.go

```diff
@@ -211,6 +211,7 @@ func startStandAlone(ctx *Context, appCreator types.AppCreator) error {
 }
 
 // legacyAminoCdc is used for the legacy REST API
+// TODO why we pass appCreator argument in startInProcess?
 func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.AppCreator) error {
```

Comment:
why we pass `appCreator` argument in `startInProcess` since it is evaluated `app := appCreator(ctx.Logger, db, traceWriter, ctx.Viper)` within `startInProcess` ?

Reply:
I'm not sure I understand. We pass `appCreator` exactly because we want to instantiate `app := appCreator()` inside startInProcess